soc/arm/st_stm32/stm32f0/dts_fixup.h

```diff
@@ -71,6 +72,34 @@
 #define DT_USART_2_IRQ				DT_ST_STM32_USART_40004400_IRQ_0
 #define DT_UART_STM32_USART_2_CLOCK_BITS	DT_ST_STM32_USART_40004400_CLOCK_BITS
 #define DT_UART_STM32_USART_2_CLOCK_BUS	DT_ST_STM32_USART_40004400_CLOCK_BUS
+#define DT_UART_STM32_USART_2_HARDWARE_FC_RTSCTS	DT_ST_STM32_USART_40004400_HARDWARE_FC_RTSCTS
+
+#define DT_UART_STM32_USART_3_BASE_ADDRESS   DT_ST_STM32_USART_40004800_BASE_ADDRESS
```

Comment:
New instances addition should be done in a separate commit (ideally a separate PR)

Reply:
I separated the modified files into several commits.